packages/selfcare-client-users-updater/src/model/UsersEventPayload.ts

```diff
@@ -0,0 +1,27 @@
+import { z } from "zod";
+
+export const EventType = z.enum(["ADD", "UPDATE"]);
+export const RelationshipStatus = z.enum(["ACTIVE", "SUSPENDED", "DELETED"]);
+
+const UserEvent = z.object({
+  userId: z.string().uuid().optional(),
+  name: z.string(),
+  familyName: z.string(),
+  email: z.string().email(),
```

Comment:
```suggestion
  email: z.string(),
```
I would parse this a simple string, since it's not on us to force that it's a correct email

Reply:
removed with commit [251476e](https://github.com/pagopa/interop-be-monorepo/pull/1789/commits/251476e8b22c69d7d42659e4eea51774c586e904)